### ISSUE TITLE
Hyper-parameter "--fuse_ab --use_syncbn" in Gold-yolo

### ISSUE THREAD

**Issue by hjg12345** [huawei-noah/Efficient-Computing#91]
I want to know about  Hyper-parameter "--fuse_ab --use_syncbn",  do you use them in your gold-yolo?  And when use them, the performance become better or worse? What is the meaning about them?

**Comment by lose4578** [comment]
Yes, we use them.
For --fuse_ab, we use it in step 1 following the YOLOv6 setting.
For --use_syncbn, we use it in anytime. Due to the version of YOLOv6 we use, training can be unstable in some cases, so we use the syncbn to stabilize the training.

**Comment by hjg12345** [comment]
Thanks for your reply and excellent work. Your code is based on yolov6-v3.0. And yolov6-v3.0's paper conduct the experiment below and I have some questions:
1. Without self-distillation, use fuse_ab without DFL
2. For self-distillation, use DFL without fuse_ab to get Teacher model, then use DFL + "--distill" to get Student model.
Is it right? And dose your experiment setting follow the above?
3. I see your gold-yolo-s.py, it has "norm_cfg=dict(type='SyncBN', requires_grad=True)", so need I use the hyperpara "--use_syncbn" when I train?
4. yolov6 uses DLD method when using self-distillation with N/S model, do you follow them at this point? And I found a small mistake in engine.py （maybe I am wrong）
![image](https://github.com/huawei-noah/Efficient-Computing/assets/127812522/a09776c5-7ce1-412b-abc6-16df8466f450)
self.cfg.model.type is "GoldYOLO-n/s", And if you follow yolov6-v3.0's DLD method, the code should change as below.
![image](https://github.com/huawei-noah/Efficient-Computing/assets/127812522/d1b40564-7c87-4fa7-bf7e-00774a816d1d)
So I hope to know more information about  your experiments including hyperparameters and more details.

**Comment by lose4578** [comment]
1. Not necessarily, depending on whether you prefer accuracy or delay. fuse_ab can improve the performance of the model, but it will also increase the delay of the model. A more detailed discussion can be found in issuse of YOLOv6.
2. Yes
3. Yes, we use syncbn for all models in ddp training mode.
4. You are right, we will update the code to fix this issue

**Comment by hjg12345** [comment]
Thanks very much
